packages/ed25519-signature-2020/src/__fixtures__/documentLoader.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  production,
+  IDocumentLoaderResponse,
+  //   development,
+} from '@transmute/jsonld-document-loader';
+
+const documentLoader = async (
+  uri: string
+): Promise<IDocumentLoaderResponse> => {
+  // uncomment to debug uris
+  // console.log(uri);
+  const _uri = uri.split('#')[0];
+
+  // needed because v3 is not published
+  if (_uri === 'https://w3id.org/security/v3') {
+    return {
+      contextUrl: null,
+      documentUrl: 'https://w3id.org/security/v3',
+      document: require('./unstable_contexts/sec-v3.json'),
+    };
+  }
+
+  // needed because v2 is not published
+  if (_uri === 'https://www.w3.org/2018/credentials/v2') {
```

Comment:
Are we mutating this context so we can issue VC's with new suites under the old context? I assumed once we had a security v3 context we would shift to creating a v3 credentials context?